src/main/java/jmgomez/apipolicy/service/PolicyService.java

```diff
@@ -1,39 +1,58 @@
 package jmgomez.apipolicy.service;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import jmgomez.apipolicy.feignClient.PolicyClient;
 import jmgomez.apipolicy.model.Accident;
-import jmgomez.apipolicy.model.dto.PolicyDto;
-import jmgomez.apipolicy.model.dto.PolicyDtoCov;
+import jmgomez.apipolicy.model.Policy;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
+import java.util.List;
+
 @Service
 public class PolicyService implements IPolicyService {
 
     private final PolicyClient policyClient;
 
+    @Autowired
+    private ObjectMapper defaultObject;
+
     @Autowired
     public PolicyService(PolicyClient policyClient) {
         this.policyClient = policyClient;
     }
 
     @Override
-    public PolicyDto[] getPolicies(String id) {
+    public List<Policy> getPolicies(String id) {
         return policyClient.getPolicies(id);
     }
 
     @Override
-    public PolicyDtoCov getPoliciesByIDs(String id)  {
-        return policyClient.getPolicyByIDs("12345" + id);
+    public Policy getPolicyByIDs(String id, String userId)  {
+        if(!("12345" + userId).equals(id)) {
```

Comment:
As we've talked through chat, you are coupling logic to what's in the mock. Very wrong decision. Your source of truth is the API, if this was an attempt to implement some authorization logic, i've sent you some links